source/LifecycleMethod.h

```diff
@@ -174,7 +201,7 @@ class LifecycleMethod {
 
   std::string base_class_name_;
   std::string method_name_;
-  std::vector<LifecycleMethodCall> callees_;
+  std::variant<std::vector<LifecycleMethodCall>,LifeCycleMethodGraph> callees_;
```

Comment:
Maybe rename this as `body_` since this represents the method body (i.e content).